src/antares/service/api_services/link_api.py

```diff
@@ -160,20 +163,57 @@ def update_link_ui(self, link: Link, ui: LinkUi) -> LinkUi:
             link_ui = LinkUi.model_validate(json_response)
 
         except APIError as e:
-            raise LinkUiUpdateError(link.name, e.message) from e
+            raise LinkUiUpdateError(link.id, e.message) from e
 
         return link_ui
 
-    def read_links(self) -> list[Link]:
-        raise NotImplementedError
+    def get_parameters(self, area_from: str, area_to: str) -> pd.DataFrame:
+        try:
+            parameters_path = f"input/links/{area_from}/{area_to}_parameters"
+            matrix = get_matrix(self._base_url, self.study_id, self._wrapper, parameters_path)
+        except APIError as e:
+            raise LinkDownloadError(f"{area_from}/{area_to}", "parameters", e.message) from e
```

Comment:
Last remark:
The exception class should expect area_form and to as parameter and do the area_from / area_to itself. This way a new user won't have to know how it's done to call the exception